docker/python3.6_ci/build_lambda.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+
+if [ -d "src" ]
```

Comment:
nit: I’d pull the `lambda_dir` variable out of the `if` block, then reference it here:

```bash
if [[ -d "$lambda_dir" ]]
```